recipe/build.bat

```diff
@@ -6,16 +6,6 @@ set "SLN_PLAT=%CMAKE_GENERATOR_PLATFORM%"
 set "SLN_TOOLSET=%CMAKE_GENERATOR_TOOLSET%"
 set "SLN_FILE=msvc\libusb.sln"
 
-if "%VS_YEAR%" == "2015" (
-  set "TGT_SDK_VERSION=10.0.14393.795"
-)
-if "%VS_YEAR%" == "2017" (
-  set "TGT_SDK_VERSION=10.0.17763.0"
-)
-if "%VS_YEAR%" == "2019" (
-  set "TGT_SDK_VERSION=10.0.20348.0"
-)
-
```

Comment:
The reason I changed this is that `win_arm64` targets vs2022 and I wasnt sure which SDK to use. I wasnt sure why this was needed, as it seems to build perfectly fine without it (I assume because this is set by the vs2022 activation script). I couldn't find any reference to a specific SDK version in the source code either so I removed it.

Reply:
I don't recall the details but this was evidently needed at some point to get the build working, but there was a time before that when it wasn't necessary either. So if it works now without it, I think it can go.